File: Harmonic_Expression.py
#a=starting term , d=difference , n=number of terms
def harmonic_progression(a,d,n):
    for i in range(0,n):
        print("1/"+ str(a+(i*d)), end = "\t")
        
a,d,n = float(input("Enter starting term")),float(input("Enter difference")),int(input("Enter number of terms"))
harmonic_progression(a,d,n)

File: Maths_And_Stats/Series/Harmonic_Progression/Harmonic_Expression.java
import java.util.Scanner;

public class Harmonic_Expression{	
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);
		long difference, startTerm, numberOfTerms;
		System.out.print("Enter Starting Term : ");
		startTerm = scan.nextLong();
		System.out.print("Enter Difference : ");
		difference = scan.nextLong();
		System.out.print("Enter Number of Terms : ");
		numberOfTerms = scan.nextLong();
		if( startTerm == 0 ){
			System.out.println("Invalid start of Harmonic Progression.");
		}
		else{
			System.out.println("Sequence is :");
			for(int i = 1; i <= numberOfTerms; i++){
				System.out.print(String.format("1/%s",(startTerm+((i-1)*difference))));
				if(i!=numberOfTerms)
					System.out.print(" , ");
			}
		}
		System.out.println();
	}
}
